libs/etterlatte-brev-model/src/main/kotlin/no/nav/etterlatte/brev/EtterlatteBrevkode.kt

```diff
@@ -7,66 +7,66 @@ Denne enumen henger tett sammen med Brevkoder-enumen.
 Vurdér om du heller bør bruke den, hvis du er utenfor rein brevbaker-kontekst.
  */
 enum class EtterlatteBrevKode(
-    val brevtype: Brevtype,
-    val tittel: String? = null, // Bruk bare denne for vedlegg. Bruk ellers tittel-feltet i Brevkoder
+    val erVedlegg: Boolean = false,
```

Comment:
I stede for å legge på denne, er det bedre å heller opprette ny enum for vedlegg? Slik at vi der har tittel i enum og slipper `tittelForVedlegg` ?

Reply:
Ja, det trur eg er fornuftig, det er jo som du seier noko litt anna. Utfordringa med det er at det er denne enumen vi no bruker i requesten mot brevbakeren, så det må litt refaktorisering til for å komme dit. Så tenkjer å ta ein titt på det i eigen PR etterpå.